lib/rubocop/cop/security/json_load.rb

```diff
@@ -0,0 +1,84 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Security
+      # This cop checks for the use of unsecure JSON methods.
+      #
+      # @example
+      #   # always offense
+      #   JSON.load("{}")
+      #   JSON.restore("{}")
+      #
+      #   # no offense
+      #   JSON.parse("{}")
+      #
+      class JSONLoad < Cop
+        DANGEROUS_METHODS = [:load, :restore].freeze
+
+        MSG = 'Do not use `JSON#%s`. Use `JSON#parse instead`.'.freeze
```

Comment:
Note that there *are* some meaningful differences between `JSON.load` and `JSON.parse`:

```
[3] pry(main)> [nil, 'null', 'true', '1', 'NaN', '"hello"'].each do |value|
[3] pry(main)*   puts %Q(JSON.parse(#{value.inspect}) => #{begin JSON.parse(value).inspect; rescue => e; e.inspect; end})  
[3] pry(main)*   puts %Q(JSON.load(#{value.inspect}) => #{JSON.load(value).inspect})
[3] pry(main)* end  
JSON.parse(nil) => #<TypeError: no implicit conversion of nil into String>
JSON.load(nil) => nil
JSON.parse("null") => #<JSON::ParserError: 784: unexpected token at 'null'>
JSON.load("null") => nil
JSON.parse("true") => #<JSON::ParserError: 784: unexpected token at 'true'>
JSON.load("true") => true
JSON.parse("1") => #<JSON::ParserError: A JSON text must at least contain two octets!>
JSON.load("1") => 1
JSON.parse("NaN") => #<JSON::ParserError: 784: unexpected token at 'NaN'>
JSON.load("NaN") => NaN
JSON.parse("\"hello\"") => #<JSON::ParserError: 784: unexpected token at '"hello"'>
JSON.load("\"hello\"") => "hello"
```

(the above is with Ruby 2.3.1)

I'm definitely in favor of adding this cop, but should we make it clearer that `JSON.parse` is not necessarily a drop-in replacement?